source/MBF.ARM.SystemCore.inc

```diff
@@ -113,7 +115,7 @@ end;
 
 function TSystemCore.ConvertMicrosecondsToCoreTicks(MicroSeconds : TMicroSeconds) : TTickCounter;
 begin
-  Result := Int64(Microseconds) * getSysTickClockFrequency div 1000000;
+  Result := (Microseconds * getSysTickClockFrequency) div 1000000;
```

Comment:
The typecast in the original was done to fix a warning, any reason that you removed it?

Reply:
My program did stall (crash) with the typecast. Without the typecast all went well. Dunno why ?